drill-cicd-common/src/main/kotlin/com/epam/drill/integration/common/agent/impl/AgentInstallerImpl.kt

```diff
@@ -98,28 +93,29 @@ class AgentInstallerImpl : AgentInstaller {
         return file
     }
 
-    override fun unzip(zipFile: File): Directory {
+    override fun unzip(zipFile: File, destDir: Directory?): Directory {
```

Comment:
Avoid using contractions - `destDir` -> `destinationDir` (`dir` is somewhat common contractions, so it is clear enough)
+ its a bit confusing with `dist`/`dest`